tools/tank_control/install_systemd_service.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Is this for running on EV3Dev or something?

Reply:
Yeah, or the Norbot. It just runs the server on boot and restarts it when it dies, so we don't have to ssh in.